### PR TITLE
fix: e2e percy snapshots

### PR DESCRIPTION
**Pull request by brian-mann**
- Closes UNIFY-1149 / https://github.com/cypress-io/cypress/issues/23435

### Details
Allows static vue assets to be accessible without going through the proxy. 

Percy generates local snapshots outside of Cypress in their own browser without configuring the proxy server. Our internal proxy server normally rejects requests made outside of the browser (in the proxy mode). This change allows static assets for vite to be served.

### CODE REVIEW

File: packages/server/lib/server-base.ts [hunk 1]
```diff
@@ -48,15 +48,39 @@ const _forceProxyMiddleware = function (clientRoute, namespace = '__cypress') {
     `/${namespace}/runner/cypress_runner.css`,
     `/${namespace}/runner/cypress_runner.js`, // TODO: fix this
     `/${namespace}/runner/favicon.ico`,
+
```
Comment (lmiller1990) [inline packages/server/lib/server-base.ts:51]: Nice, good solution

**Review by lmiller1990 — DISMISSED** [review]
Seems fine - good solution. For anyone else looking at this PR - there's a ton of renames here, the main change is in this file: https://github.com/cypress-io/cypress/pull/20604/files#diff-040b9d41ce1efd9534f52c69af25d3d587c3a5b9cc9cb8b39a2848ec3e7e420e.

I also pulled the branch locally and tested a bit, everything seems :100: . We had better approve the remaining 18 snapshots on Percy before merging (since they are now showing correctly).

File: packages/server/lib/server-base.ts [hunk 2]
```diff
@@ -48,15 +48,39 @@ const _forceProxyMiddleware = function (clientRoute, namespace = '__cypress') {
...
+    // the reason this is not namespace is because we don't launch the vite
```
Comment (ryanthemanuel) [inline packages/server/lib/server-base.ts:52]:
```suggestion
    // the reason this is not namespaced is because we don't launch the vite
```

**Review by ryanthemanuel — DISMISSED** [review]

File: packages/app/vite.config.ts [hunk 3]
```diff
@@ -1,10 +1,9 @@
 import { makeConfig } from '../frontend-shared/vite.config'
 import Layouts from 'vite-plugin-vue-layouts'
 import Pages from 'vite-plugin-pages'
-import Copy from 'rollup-plugin-copy'
```
Comment (elevatebart) [inline packages/app/vite.config.ts:4]: Thank you for that !! 
Great clean-up.

Should we also remove the plugin from the dependencies in `package.json`?

**Comment by tbiethman** [comment]
@brian-mann Is this PR ready for review? I know you are working on cleaning up some of the snapshots, but I'm not sure if you are wanting to merge those changes here or in a follow-up PR.

File: packages/app/vite.config.ts (hunk 3 above)
Comment (lmiller1990) [reply packages/app/vite.config.ts:4]: Makes sense, I went ahead and removed it.

**Comment by lmiller1990** [comment]
I went ahead and merged in 10.0-release, resolved conflicts, and made the trivial review changes (typos, remove unused dep from `package.json`.

@tbiethman it's marked as ready and in review - I'd assume we are good to review/merge.

**Review by lmiller1990 — APPROVED** [review]

**Comment by jennifer-shehane** [comment]
I confirmed with @brian-mann - he said this should still be in draft, he has some things to finalize. @lmiller1990 if you'd like you can sync with Brian to see what he wanted done and see if you can take over the tasks. Otherwise it's not ready for review.

**Review by jennifer-shehane — CHANGES_REQUESTED** [review]
@brian-mann Update the base branch to be against develop before merging.

**Comment by emilyrohrbough** [comment]
Closing as stale. We can re-visit at a later time.

**Pull request closed without merging**